polkadot-sdk/substrate/client/consensus/grandpa/Cargo.toml

```diff
@@ -0,0 +1,60 @@
+[package]
```

Comment:
It might be better to use git submodule and put the patches in a seperate repo so that we can clearly see what have been patched.